miso-web/src/main/webapp/scripts/bulk_sample.js

```diff
@@ -587,6 +667,19 @@ BulkTarget.sample = (function ($) {
                       })(),
                   disabled: true,
                 });
+              } else if (projectSelected && projectSelected.assayIds !== null) {
+                var assayLst = [];
+                for (var i = 0; i < projectSelected.assayIds.length; i++) {
+                  var anAssay = Constants.assays.find(function (x) {
+                    return x.id === projectSelected.assayIds[i];
+                  });
+                  assayLst.push(anAssay);
+                }
+                api.updateField(rowIndex, "requisitionAssayId", {
+                  source: assayLst,
+                  value: null,
+                  disabled: false,
+                });
```

Comment:
* duplicated code - move to a function
* array and for loop are unnecessary. Can reduce to

    ```
    source: projectSelected.assayIds.map(function(assayId) {
        return Utils.array.findUniqueOrThrow(Utils.array.idPredicate(assayId), Constants.assays);
    }),
    ```